rudder-lang/tests/tester.sh

```diff
@@ -8,7 +8,7 @@ cfjson_tester=$PWD/tests/helpers/cfjson_tester
 mkdir -p $dir/target/
 
 # Take original technique an make a json
-$cfjson_tester ncf-to-json $dir/cf/${name}.cf $dir/json/${name}.json
+$cfjson_tester ncf-to-json $dir/cf/${name}.cf $dir/json/${name}.json /tools/rudderc-dev.conf
```

Comment:
relative file path should nos start with a /

Reply:
This file has been totally rewritten since this commit